Classes/Systems/Secrets.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Secrets.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 23/11/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+enum Secrets {
+    
+    enum GitHub {
+        static let clientId = Secrets.environmentVariable(named: "GITHUB_CLIENT_ID")
+        static let clientSecret = Secrets.environmentVariable(named: "GITHUB_CLIENT_SECRET")
+    }
+    
+    enum Imgur {
+        static let clientId = Secrets.environmentVariable(named: "IMGUR_CLIENT_ID")
+    }
+    
+    fileprivate static func environmentVariable(named: String) -> String {
+        
+        let processInfo = ProcessInfo.processInfo
+        
+        guard let value = processInfo.environment[named] else {
+            print("‼️ Missing Environment Variable: '\(named)'")
```

Comment:
We could do `#if DEBUG then fatalError()` if we wanted to be super obvious when doing things locally?